mapbox/services/tokens.py

```diff
@@ -23,8 +23,73 @@ def create(self, username, scopes=None, note=None, expires=0):
         payload = {'scopes': scopes, 'note': note}
 
         if expires > 0:
-            payload['expires'] = arrow.now().replace(seconds=+expires).isoformat()
+            payload['expires'] = (datetime.now() + timedelta(seconds=expires)).isoformat()
 
         res = self.session.post(uri, json=payload)
         self.handle_http_error(res)
         return res
+
+    def list_tokens(self, username, limit=None):
+        uri = URITemplate(self.baseuri + '/{username}').expand(username=username)
+
+        params = {}
+        if limit:
+            params['limit'] = int(limit)
+
+        res = self.session.get(uri, params=params)
+        self.handle_http_error(res)
+        return res
+
+    def create_temp_token(self, username, scopes=None, expires=0):
```

Comment:
Haha, true. I thought it was anyway going to be handled by the condition `0 <expires<=3600`. However, I have updated it to 3600 with other changes.